test/parsing.jl

```diff
@@ -51,6 +53,8 @@
                     "--name", "myplayground",
                     "--julia-version", "julia-0.3",
                     "--requirements", "/path/to/requirements",
+                    "--julia-registry", "https://github.com/rofinn/Playground.jl",
```

Comment:
This seems like a weird test since Playground.jl isn't a registry. Might want to change this to "https://github.com/rofinn/METADATA.jl"?

Reply:
Yeah I just used the first url that came to mind since it's only parsing the arguments and not doing anything with them. Changed now.